packages/backend/src/core/balances/BalanceUpdater.test.ts

```diff
@@ -298,3 +307,17 @@ describe(BalanceUpdater.name, () => {
     }
   }
 })
+function mockBalance(
+  assetId: AssetId,
+  timestamp: UnixTime,
+  holderAddress: EthereumAddress,
+  chainId: ChainId,
+): BalanceRecord {
```

Comment:
Nit pick: mockBalance always get the same timestamp, so it can be ommited as an argument here